codes/extractive/netsum/netsum.py

```diff
@@ -206,6 +210,9 @@ def get_sent_word_sens(sentence):
     for w in re.findall(r'[a-zA-Z]+', sentence):
         if w in STOP_WORDS or len(w) < WORD_MIN_LEN:
             continue
+        # skip words that are deemed to be unimportant by the IDF score.
+        if with_idf_focus and news_words.get(w) < IDF_THRESHOLD:
```

Comment:
would it be better to use a tf_idf_threshold here? so we consider tf as well.

Reply:
Yeah it's just a naming convention. Basically here it's trying to neglect words that have a low IDF score.